generated/niswitch/session.py

```diff
@@ -181,11 +181,8 @@ class _SessionBase(object):
     '''Type: str
 
     This property indicates the Driver Setup string that the user  specified when initializing the driver.
-    Some cases exist where the end-user must specify instrument driver  options at initialization time.  An example of this is specifying  a particular instrument model from among a family of instruments  that the driver supports.  This is useful when using simulation.   The end-user can specify driver-specific options through  the DriverSetup keyword in the optionsString parameter to the  InitWithOptions method, or through the IVI Configuration Utility.
+    Some cases exist where the end-user must specify instrument driver  options at initialization time.  An example of this is specifying  a particular instrument model from among a family of instruments  that the driver supports.  This is useful when using simulation.   The end-user can specify driver-specific options through  the DriverSetup keyword in the optionsString parameter to the  init_with_options method, or through the IVI Configuration Utility.
     If the user does not specify a Driver Setup string, this property returns an empty string.
-
-    Note:
-    One or more of the referenced methods are not in the Python API for this driver.
```

Comment:
This note was useful and now it's gone. That's a problem.

Reply:
See comment above. The note will come back when the metadata is updated without `'codegen-method': 'no'` functions.